### PR TITLE
[1807] Bugfix - hide published-on if never published

### PR DESCRIPTION
**Pull request by timabell**
Was falling over with a nil error for new records.

### CODE REVIEW

**Comment by timabell** [comment]
[[S] accredited body (pull onto Rails), frontend](https://trello.com/c/ATAPQ9c6/1807-s-accredited-body-pull-onto-rails-frontend)

File: app/views/providers/details.html.erb [hunk 1]
```diff
@@ -71,7 +71,9 @@
       <div class="govuk-!-margin-bottom-6" data-qa="provider__content-status">
         <%= provider.status_tag %>
       </div>
-      <p class="govuk-body">Last published on <%= l(@provider&.last_published_at.to_date) %></p>
+      <% if @provider&.last_published_at %>
```
Comment (defong) [inline app/views/providers/details.html.erb]: where the test?

**Comment by defong** [comment]
also ain't it this https://trello.com/c/9XGmLrD3/1921-s-handle-brand-new-provider-enrichment

**Comment by timabell** [comment]
As @defong helpfully pointed out there are no tests. Nothing failed with or without this patch so I'll be adding some coverage before merging.

**Comment by timabell** [comment]
Damn, typo in branch name, going to have to close this one and try again. I'll hold of opening another till the tests are ready

**Pull request closed without merging**